src/com/ansorgit/plugins/bash/editor/inspections/inspections/ExtendedTestCommandVisitor.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) Joachim Ansorg, mail@ansorg-it.com
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ansorgit.plugins.bash.editor.inspections.inspections;
+
+import com.ansorgit.plugins.bash.editor.inspections.quickfix.DoubleBracketsQuickfix;
+import com.ansorgit.plugins.bash.lang.psi.BashVisitor;
+import com.ansorgit.plugins.bash.lang.psi.api.shell.BashConditionalCommand;
+import com.intellij.codeInspection.ProblemsHolder;
+
+public class ExtendedTestCommandVisitor extends BashVisitor {
```

Comment:
I'd prefer a self-contained inspection. Could you make this visitor a static inner class of `UseExtendedTestCommandInspection`?